install_files/ansible-base/roles/app/tasks/initialize_securedrop_app.yml

```diff
@@ -141,6 +141,15 @@
     - gpg
     - securedrop_config
 
+- name: Copy custom notification text to config.py.
+  lineinfile:
+     dest: "{{ securedrop_code }}/config.py"
+     regexp: "^CUSTOM_NOTIFICATION"
+     line: "CUSTOM_NOTIFICATION = '{{ custom_notification_text }}'"
+  when: custom_notification_text is defined and custom_notification_text != ''
```

Comment:
Good idea @fowlslegs: I have added `custom_notification` text to the development and staging VMs